### PR TITLE
ch_checker: Add copyright check for yml files

### LINKED ISSUE #6: Improvement: Add copyright header for all .yml files.
> [!IMPORTANT]
> Make sure to link this issue with the PR for your improvement.

In current setup, all .yml files missing copyright header info. There are 2 parts of improvement.
1. Set the copyright header info in yml files.
2. Enable cr_checker tool to verify .yml files.

### PR DESCRIPTION
**Pull request by nradakovic**
All source files need to have a copyright header based on the Eclipse Foundation handbook. YML files are source files for GitHub Actions and Workflows. In addition to adding a copyright header to each YML file, the cr_checker will now check for their existence in every new YML file that is added to the repository.

Issue-ref: resolves #6

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -21,25 +23,29 @@ alias(
     actual = "//tools/format:format.fix",
 )
 
-alias(
+checker(
     name = "copyright.check",
```
Comment (ltekieli) [inline BUILD:27]: .check and .fix will always be instantiated with the same input, best to hide those under the one macro.

File: tools/cr_checker/cr_checker.bzl [hunk 2]
```diff
@@ -13,120 +13,72 @@
...
-        "log_file": attr.bool(mandatory = False, default = False),
-        "use_memory_map": attr.bool(mandatory = False, default = False),
-        "_script": attr.label(
-            default = Label("//tools/cr_checker/tool:cr_checker"),
-        ),
-    },
-)
-
-def check(
+def checker(
```
Comment (ltekieli) [inline tools/cr_checker/cr_checker.bzl:16]: Can you do cr_checker, or even better copyright_checker.

File: BUILD (hunk 1 above)
Comment (nradakovic) [reply BUILD]: I'm not sure if I understood what you want. Should the macro expand both targets? Check the update; if this is not what you want, let me know.

File: tools/cr_checker/cr_checker.bzl (hunk 2 above)
Comment (nradakovic) [reply tools/cr_checker/cr_checker.bzl]: Done.

File: BUILD (hunk 1 above)
Comment (ltekieli) [reply BUILD]: Yes, exactly, thanks.

**Review by ltekieli — APPROVED** [review]

**Pull request merged**